web_access_rule_buttons/models.py

```diff
@@ -15,10 +15,17 @@ def check_access_rule_all(self, operations=None):
    :param operation: a list of ``read``, ``create``, ``write``, ``unlink``
    :return: {operation: access} (access is a boolean)
     """
+    self.ensure_one()
```

Comment:
Do you need this? Why not simply check `self.ids` and this continues being multi-record?